libs/platform/auth/src/components/login/login.component.ts

```diff
@@ -55,7 +55,8 @@ export class AuthLoginComponent extends ContentMixin<LoginOptions>(LitElement) {
     }),
     switchMap((request) =>
       this.authLoginStrategy.login(request).pipe(
-        catchError(() => {
+        catchError((e) => {
+          console.error(e);
```

Comment:
Do we need it?

Reply:
@gund Recommended it to have more useful error messages in case the login fails.

Reply:
Had a closer look. We don't currently have notification center in fulfillment app. Also, we catch the error, so even if notification center was set up, it will only display uncaught errors. If we really wanted to tie it to a notification center, then we might use the ErrorService directly for that.